File: Problems/Tarifa/Tarifa.cpp
#include <iostream>
#include <vector>
using namespace std;

int main() {
	int x, n, total, temp;

	cin >> x >> n;

	total = x * (n + 1);

	for (int i = 0; i < n; i++) {
		cin >> temp;
		total -= temp;
	}

	cout << total;

	return 0;
}

File: Problems/Tarifa/Tarifa.go
package main

import(
    "fmt"
)

func main(){
    var x, n, temp int
    
    fmt.Scan(&x)
    fmt.Scan(&n)
    
    total := x * (n + 1)
    
    for i := 0; i < n; i++{
        fmt.Scan(&temp)
        total -= temp
    }
    
    fmt.Println(total)
}
